src/Monolog/ResettableInterface.php

```diff
@@ -27,5 +27,10 @@
  */
 interface ResettableInterface
 {
+    /**
+     * Function that executes the resetting of a Processor
+     *
+     * @return void
```

Comment:
I wanted to create another PR adding void return type to reset function but I guess this would mean a huge BC for symfony... right ? ping @lyrixx

Reply:
In Symfony, we decided to not add `void` as return type to make it easier to change our mind later if needed.

Reply:
I will revert then...

Reply:
I decided not to add it as it's brand new in 1.x and that would mean breaking new implementations that could otherwise be compatible with 1.x and 2.x. Didn't really seem worth it. I'm fine having it in the docblock though for clarity.